### PR TITLE
Story2

### PR DESCRIPTION
**Pull request by JamesMcGraaw**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,44 @@
-import './App.css';
-import Footer from "./components/Footer";
-import RecentJobs from "./components/RecentJobs";
-import Header from "./components/Header";
+import './App.css'
```
Comment (nealioa) [inline src/App.js]: Consistency: single or double quotes?

File: src/components/DetailModal/index.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+const DetailModal = ({data, setData, skills, setSkills, selectedID}) => {
+    return (
+        <>
+            <div className="modal fade" id="jobDetails" tabIndex="-1" aria-labelledby="jobDetailsLabel"
+                 aria-hidden="true">
+                <div className="modal-dialog modal-lg">
+                    <div className="modal-content">
+                        <div className="modal-header">
+                            <p className="modal-title fs-6" id="exampleModalLabel">
+                                <strong>{data.job_title} - {data.company}</strong></p>
```
Comment (nealioa) [inline src/components/DetailModal/index.js]: Is the `<strong>` tag actually necessary?

To quote the MDN docs:
> The `<strong>` element is for content that is of "strong importance," including things of great seriousness or urgency (such as warnings).

Is the bold look purely a style thing (in which case Bootstrap's `fw-bold` would do it) or should a screen reader be stressing the title and company to the listener?

File: src/components/DisplayJobs/index.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                        Skills
+                    </th>
+                </tr>
+                </thead>
+                <tbody>
+                    {displayJobs.map((job, key) => {
+                        const roundedSalary = job.salary ? Number(job.salary.toPrecision(2)) : ("")
+                        return (
+                            <tr key={key}>
+                            <td scope="row">
```
Comment (nealioa) [inline src/components/DisplayJobs/index.js]: Consistency: The following content should be indented within the `<tr>`.

File: src/components/DisplayJobs/index.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                            {window.innerWidth > 768 && (<img className="block" width="100" src={job.logo} alt="logo"/>)}
+                            </td>
+                            <td>
+                                <p
+                                className="h4 jobLink"
```
Comment (nealioa) [inline src/components/DisplayJobs/index.js]: Consistency: Other tags with multiple attributes on separate lines have them indented.

File: src/components/DisplayJobs/index.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                                >
+                                {job.job_title}
+                                </p>
+                                <p>{job.company}</p>
+                            </td>
+                            <td className="row-span-2">
+                                {job.type ? (<button className="btn btn-primary">{job.type}</button>) : ""}
+                            </td>
+                            <td className="row-span-2">
+                            {roundedSalary ? `£${roundedSalary}` : ""}
```
Comment (nealioa) [inline src/components/DisplayJobs/index.js]: Consistency: Indent.

File: src/components/DisplayJobs/index.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+                            </td>
+                            <td className="row-span-2">
+                            {job.skills.map((skill, index) => {
```
Comment (nealioa) [inline src/components/DisplayJobs/index.js]: Consistency: Indent.

File: src/index.js [hunk 7]
```diff
@@ -7,7 +7,7 @@ import reportWebVitals from './reportWebVitals';
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <App  data-bs-toggle="modal" data-bs-target="#exampleModal"/>
```
Comment (nealioa) [inline src/index.js]: exampleModal? Really? Surely we can think of a more appropriate name.

**Review by nealioa — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (JamesMcGraaw) [reply src/App.js]: Made executive decision - all singles

Comment (JamesMcGraaw) [reply src/App.js]: Scratch that - all doubles.

File: src/components/DetailModal/index.js (hunk 2 above)
Comment (JamesMcGraaw) [reply src/components/DetailModal/index.js]: Changed this with fw-bold

File: src/components/DisplayJobs/index.js (hunk 3 above)
Comment (JamesMcGraaw) [reply src/components/DisplayJobs/index.js]: Done

File: src/components/DisplayJobs/index.js (hunk 4 above)
Comment (JamesMcGraaw) [reply src/components/DisplayJobs/index.js]: Done

File: src/components/DisplayJobs/index.js (hunk 5 above)
Comment (JamesMcGraaw) [reply src/components/DisplayJobs/index.js]: done

File: src/components/DisplayJobs/index.js (hunk 6 above)
Comment (JamesMcGraaw) [reply src/components/DisplayJobs/index.js]: Done

File: src/index.js (hunk 7 above)
Comment (JamesMcGraaw) [reply src/index.js]: Redundant code. Removed

**Review by nealioa — DISMISSED** [review]

**Review by nealioa — APPROVED** [review]

**Pull request merged**